modules/server/prisma.ts

```diff
@@ -1,10 +1,11 @@
 import {PrismaClient} from "@prisma/client";
 
-const {DB_1_PORT_27017_TCP_ADDR} = process.env;
+var config = require('../../config/config');
+
 export const prisma = new PrismaClient({
     datasources: {
         db: {
-            url: `mongodb://${DB_1_PORT_27017_TCP_ADDR}:27017/mean-dev`
+            url: config.db.uri
```

Comment:
this is not exactly the same, looks like the port 27017 isn't included, but it seems to default to the correct one
```
    uri: process.env.MONGOHQ_URL || process.env.MONGOLAB_URI || 'mongodb://' + (process.env.DB_1_PORT_27017_TCP_ADDR || 'localhost') + '/mean',
```

Reply:
This looks correct, I think this fixes the issue that Davy found in prod, specifically because it doesn't add another 27017 at the end of the string.

But it does mean that the config in the `compose.yaml` is wrong - that's where I was misled.

Can you update `compose.yaml` and `compose-prod.yaml` to set DB_1_PORT_27017_TCP_ADDR to `mongodb:27017`?

I think the reason why nothing is broken is because 27017 is the default mongodb port, so it's probably ok to either specify it or leave it out of the connection uri.